openedx_webhooks/info.py

```diff
@@ -7,6 +7,7 @@
 from typing import Dict, Iterable, Optional, Union
 
 import yaml
+import csv
```

Comment:
nit: stdlib imports should be in the section above, and in alphabetical order (before "import datetime")

Reply:
Moved!